src/numberAxis/Funnel.tsx

```diff
@@ -56,9 +54,12 @@ interface FunnelProps extends InternalFunnelProps {
   animationDuration?: AnimationDuration;
   animationEasing?: AnimationTiming;
   id?: string;
+  trapezoids?: FunnelTrapezoidItem[];
+  animationId?: number;
+  activeTooltipIndex?: number;
```

Comment:
So, if I understand this correctly, we are adding the activeTooltipIndex (and thereby increase complexity of the API) for the reason of not breaking existing implementations. 

The alternative would be to simply set the activeIndex internally, which is already exposed. 

How about only overwriting the activeIndex if it is not set externally, without exposing the activeTooltipIndex? Would that work?

Reply:
I am a bit hesitant to introduce the activeTooltipIndex, because I find it complicates the API further (which is already not trivial). I would prefer if we found a way that could not extend the API, but yet reach feature parity.

Reply:
What I find interesting in this context is that activeIndex is only exposed in a few cases: Funnel, Scatter, RadialBar, Pie. 

The decision we make here will carry weight downstream, as we unify the API further.

Reply:
> How about only overwriting the activeIndex if it is not set externally, without exposing the activeTooltipIndex? Would that work?

> I am a bit hesitant to introduce the activeTooltipIndex, because I find it complicates the API further (which is already not trivial). I would prefer if we found a way that could not extend the API, but yet reach feature parity.

Yes, this all makes sense to me. Just pushed up that change.